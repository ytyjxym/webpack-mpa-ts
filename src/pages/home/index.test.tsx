import React from 'react'
import Home from './index'
import renderer from 'react-test-renderer'
test('test', () => {
    const component = renderer.create(
        <Home></Home>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})