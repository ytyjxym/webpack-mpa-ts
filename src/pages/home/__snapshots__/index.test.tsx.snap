// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
Array [
  <div
    className="heade"
  >
    1
    2
    3
  </div>,
  <button
    className="ant-btn ant-btn-danger"
    onClick={[Function]}
    type="button"
  >
    <span>
      1223
    </span>
  </button>,
  "1",
  "2",
  "3",
]
`;
